Casks/l/logos.rb

```diff
@@ -13,7 +13,9 @@
 
   livecheck do
     url "https://clientservices.logos.com/update/v1/feed/logos10-mac/stable.xml"
-    regex(%r{<logos:version[^>]*>(\d+(?:\.\d+)+)</logos:version>}i)
+    strategy :xml do |xml|
+      xml.get_elements("//link").map { |item| item.attributes["logos:version"] }
```

Comment:
```suggestion
      xml.get_elements("//logos:version")&.map { |item| item.text&.strip }
```

For what it's worth, the existing regex is checking elements like `<logos:version>36.1.0.0006</logos:version>`, not the `logos:version` attribute of `link` elements (e.g., `<link href='https://downloads.logoscdn.com/LBS10/Installer/36.1.0.0006/LogosMac.dmg' logos:version='36.1.0.0006' />`). They both provide the same value but this suggestion would maintain the existing behavior (unless there's a benefit to checking the `link` attribute).

Reply:
Makes sense, thank you!